src/Tests/BingSearch.fs

```diff
@@ -38,7 +38,7 @@ let tests = testList "Bing Search" [
         let model = baseArm.JsonModel |> convertTo<{| kind: string |}>
         Expect.equal bsArm.Name (ResourceName "test") "Name"
         Expect.equal bsArm.Location Location.WestEurope "Location"
-        Expect.isFalse bsArm.Properties.statisticsEnabled "Statistics enabled"
+        Expect.equal bsArm.Statistics Disabled "Statistics not enabled"
```

Comment:
Revert back to `isFalse`; there's a member on the Enabled / Disabled called `toBool` or something.

Reply:
But `equal Statistics Disabled` isn't more meaningful than `isFalse Statistics`?

Reply:
hmmm. I thought:

```fsharp
Expect.isFalse bsArm.Statistics.AsBoolean "Statistics not enabled"
```

That's what the `isFalse` function is for - to test against booleans. I suppose I'm thinking of Enable / Disabled as a bool here, but maybe you're right.

The alternative (maybe preferably) would be to pull at the JSON boolean value into `model` (like you've done with `kind`) and compare to the boolean directly. That's actually my preference for all tests wherever possible because that then tests the actually emitted JSON.

Reply:
@isaacabraham I updated the JSON model to include also a `statisticsEnabled` field and I test it against `isFalse`/`isTrue`. I keep also a check for `bsArm.Statistics`.